RadarSDK/RadarLocationManager.m

```diff
@@ -337,8 +346,11 @@ - (void)updateTracking:(CLLocation *)location {
         } else {
             [self stopUpdates];
             [self removeAllRegions];
-            [self.locationManager stopMonitoringVisits];
-            [self.locationManager stopMonitoringSignificantLocationChanges];
+            
+            if (!fromInitialize) {
```

Comment:
would it make sense to flip the function argument? For example, the signature could become

```objective-c
- (void)updateTracking:(CLLocation *)location
        withStandardWorkflow:(BOOL)isStandardWorkflow
```

Then the default implementation would pass `YES` and the initialize workflow would pass `NO`, and this if check could be flipped. It's a really small optimization, but could also help with the function naming.